subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/LazyDownloadsIntegrationTest.groovy

```diff
@@ -107,10 +107,11 @@ class LazyDownloadsIntegrationTest extends AbstractHttpDependencyResolutionTest
         failure.assertHasCause("Could not resolve test:test:1.0.")
 
         where:
-        expression                                | _
-        "files"                                   | _
-        "fileCollection { true }"                 | _
-        "resolvedConfiguration.resolvedArtifacts" | _
-        "incoming.artifacts"                      | _
+        expression << [
```

Comment:
I am not sure you understood my original comment. This variable is not used in this test.

Reply:
Oh, I see.  I did misunderstand.  I'll take another look at this.